damus.xcodeproj/project.pbxproj

```diff
@@ -738,11 +739,12 @@
 			};
 			buildConfigurationList = 4CE6DEDE27F7A08100C66700 /* Build configuration list for PBXProject "damus" */;
 			compatibilityVersion = "Xcode 13.0";
-			developmentRegion = en;
+			developmentRegion = "en-US";
 			hasScannedForEncodings = 0;
 			knownRegions = (
-				en,
 				Base,
+				"es-419",
```

Comment:
`es-419` is Spanish (Latin America). That should hopefully cover a large enough market. If there are differences in translations for specific versions of Spanish within Latin American regions, we can create target locales for those too.